src/prospectScraper/ProspectScraper.cs

```diff
@@ -0,0 +1,625 @@
+            //" May 21, 2019 2:00 AM EST"
+            string dateInNiceFormat = parsedDate.ToString("yyyy-MM-dd");
+
+            List<ProspectRanking> list1 = GetProspects(document1, parsedDate, 1);
+            List<ProspectRanking> list2 = GetProspects(document2, parsedDate, 2);
+            List<ProspectRanking> list3 = GetProspects(document3, parsedDate, 3);
+            List<ProspectRanking> list4 = GetProspects(document4, parsedDate, 4);
+            List<ProspectRanking> list5 = GetProspects(document5, parsedDate, 5);
+
+            //This is the file name we are going to write.
+            var csvFileName = $"ranks{Path.DirectorySeparatorChar}{dateInNiceFormat}-ranks.csv";
+
+            Console.WriteLine("Creating csv...");
+
+            //Write projects to csv with date.
+            using (var writer = new StreamWriter(csvFileName))
+            using (var csv = new CsvWriter(writer, CultureInfo.CurrentCulture))
+            {
+                csv.Configuration.RegisterClassMap<ProspectRankingMap>();
+                csv.WriteRecords(list1);
+                csv.WriteRecords(list2);
+                csv.WriteRecords(list3);
+                if (list4.Count > 0)
+                {
+                    csv.WriteRecords(list4);
+                }
+                if (list5.Count > 0)
+                {
+                    csv.WriteRecords(list5);
+                }
+            }
+
+            CheckForMismatches(csvFileName);
+            CreateCombinedCSV();
+            CheckForMismatches($"ranks{Path.DirectorySeparatorChar}combinedRanks2021.csv");
+            CreateCombinedCSVWithExtras();
+
+            Console.WriteLine("Big Board Completed.");
+        }
+
+        public void RunTheMockDraft(bool parseDate)
+        {
+            //TODO - Implement Mock Draft
+            var webGet = new HtmlWeb
+            {
+                UserAgent = "Mozilla/5.0 (Windows NT 6.3; WOW64; rv:31.0) Gecko/20100101 Firefox/31.0"
+            };
+            var document1 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-1.asp");
+            var document3 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-2.asp");
+            var document4 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-3.asp");
+            var document5 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-4.asp");
+            var document6 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-6.asp");
+
+            // Need to get date of mock draft eventually.
+            HtmlNode hn = document1.DocumentNode;
+            HtmlNode hi1 = hn.SelectSingleNode("//*[@id='HeadlineInfo1']");
+            DateTime mockDraftDate = ChangeDateStringToDateTime(hi1.InnerText.Replace(" EST", "").Trim(), parseDate);
+            string draftDate = FormatScrapedDate(mockDraftDate);
```

Comment:
This `FormatScrapedDate` method can be replaced with:
`string draftDate = draftDate.ToString("yyyy-MM-dd");`